tests/unit/test_urls.py

```diff
@@ -38,21 +38,21 @@ def test_unc_path_to_url_win() -> None:
 
 @pytest.mark.skipif("sys.platform != 'win32'")
 def test_relative_path_to_url_win() -> None:
-    resolved_path = os.path.join(os.getcwd(), "file")
-    assert path_to_url("file") == "file:" + urllib.request.pathname2url(resolved_path)
+    path = os.path.join(os.getcwd(), "file").replace("\\", "/")
+    assert path_to_url("file") == "file:///" + urllib.parse.quote(path, safe="/:")
```

Comment:
Just to be sure, `file:c:/file` (and `file:/c:/file`) is the non-normative form of `file:///c:/file` but they are effectively equivalent, right?

Reply:
Also, on Windows, does `pathname2url()` also include `:` in its url-safe default list? I'm on Linux, and `pathname2url("C:\\Program Files")` returns `file:C%3A%5CProgram%20Files` which is fine for Linux, but would be nonsensial for Windows.